src/status_im2/contexts/debug_component/view.cljs~

```diff
@@ -0,0 +1,16 @@
+(ns status-im2.contexts.debug.view
+  (:require [react-native.core :as rn]
+            [utils.re-frame :as rf]))
+
+(def ^:private container
+  {:flex               1
+   :padding-horizontal 20
+   :justify-content    :center})
+
+(defn view
+  []
+  (let [debug-view (rf/sub [:debug/component])]
+    [rn/view
+     {:style container}
+     (when (not (nil? debug-view))
+       debug-view)]))
```

Comment:
Why do we need this file?

Reply:
It's a backup file generate automatically. Will be removed I'm sure.

Reply:
my bad, didn't notice the tilda